Assignment/solution_student/index.html

```diff
@@ -1,35 +1,38 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+    <title>Joe Johnson</title>
+    <link rel="stylesheet" href="style.css">
+</head>
 
-Sample Content
-——-
+<header>
 
-ABOUT ME PAGE
+<h2><em>About Me</em> | <a href="resume.html">Porfolio</a></h2>
+<h1>Joe Johnson</h1>
 
-I ENJOY LIFE AS A DEVELOPER
 
-I'm Joe Johnson, a Developer based in NYC. I Have ten years of experience in the graphic design world, specializing in the creation of responsive websites.
+</header>
 
+<body>
+<hr>
 
-RESUME PAGE
+<div id="bio">
+		<h3>I Enjoy Life as Developer</h3>
 
-Experience
-        
-I recently graduated from a Front-End Web Development course at General Assembly where I learned HTML, CSS, JavaScript, jQuery and how to be an awesome Front-End Web Developer! During my spare time, I enjoy senior water aerobics, pickling, and spending time with my goldfish, Jerry.
-    
-Relevant Experience
-    
-Web Developer, Relaxr 2015 - present
-Developed a multi-column layout blog, landing page, and contact forms that render on mobile devices.
+		<p>I'm Joe Johnson, a Developer based in NYC. I Have ten years of experience in the graphic design world, specializing in the creation of responsive websites.</p>
+	</div>
+<div id="photo">
+		<img src="images/avatar.png" alt="Joe Johnson Photo" />
+	</div>
+</div>
 
-Web Developer, Startup Matchmaker 2015 - present
-Used a design team's wireframes to develop this company's responsive homepage.
+<div style="clear: both;"></div>
```

Comment:
Very good on figuring out you need to apply `clear: both` to something down here, but remember that we don't **ever** want to apply styles _inline_ with the "style" attribute — instead do this in your CSS document by giving that div a specific `class`, or apply the `clear: both` to the `<footer>` perhaps